src/MailTrackerController.php

```diff
@@ -44,19 +43,15 @@ public function getT($hash)
         return $response;
     }
 
-    public function getL($url, $hash)
+    public function getN(Request $request)
     {
-        $url = base64_decode(str_replace("$", "/", $url));
+        $url  = $request->l;
+        $hash = $request->h;
+
         if (filter_var($url, FILTER_VALIDATE_URL) === false) {
-            throw new BadUrlLink('Mail hash: '.$hash.', URL: '.$url);
+            throw new BadUrlLink('Mail hash: ' . $hash . ', URL: ' . $url);
```

Comment:
This was only present in the getL method, but moved it to getN so that the docs are accurate